<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nota_label1</class>
 <widget class="QMainWindow" name="nota_label1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IF Média</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A colocar imagem&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="aviso">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;*Notas vão de 0 a 100&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="notalabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nota 1: </string>
    </property>
   </widget>
   <widget class="QLabel" name="notalabel2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nota 2: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="tamanho_materia">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2 bimestres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 bimestres</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="materia">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matéria de: </string>
    </property>
   </widget>
   <widget class="QLabel" name="notalabel3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nota 3: </string>
    </property>
   </widget>
   <widget class="QLabel" name="notalabel4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nota 4: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nota1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nota2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nota3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nota4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="resultado">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
   <widget class="QLabel" name="situacao_l">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Situação: </string>
    </property>
   </widget>
   <widget class="QLabel" name="situacao_l_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Media final:</string>
    </property>
   </widget>
   <widget class="QLabel" name="situacao_l_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prova final: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="situacao">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="media_final">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="prova_final">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>570</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="calcular">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>392</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
